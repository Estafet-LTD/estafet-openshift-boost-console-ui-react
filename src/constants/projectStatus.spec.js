import PROJECT_STATUS from './projectStatus';

describe('Constants: projectStatus', () => {
  it('should match the snapshot', () => {
    expect(PROJECT_STATUS).toMatchSnapshot();
  });
});
