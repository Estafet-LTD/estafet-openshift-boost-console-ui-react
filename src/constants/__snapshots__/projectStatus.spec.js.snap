// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: projectStatus should match the snapshot 1`] = `
Object {
  "ACTIVE": "Active",
  "TERMINATING": "Terminating",
}
`;
